sources/ClangSharp.PInvokeGenerator/PInvokeGeneratorConfiguration.cs

```diff
@@ -54,13 +50,32 @@ public PInvokeGeneratorConfiguration(string libraryPath, string namespaceName, s
             MethodPrefixToStrip = methodPrefixToStrip;
             Namespace = namespaceName;
             OutputLocation = outputLocation;
-            RemappedNames = remappedNames;
+
+            _remappedNames = new Dictionary<string, string>()
```

Comment:
can this be made config?

Reply:
The current hardcoded list represents 4 types defined by the C/C++ standard to be platform sized, so this gives you the correct behavior by default for common scenarios.

You can also override these with new values if you wanted to customize them. -- The only thing you can't do is resolve them to the base typedef via "normal" logic, which is what a switch would allow.

I can certainly add a switch (maybe `no-default-remap` or something).